webclient/dokument/models.py

```diff
@@ -135,7 +135,12 @@ class Dokument(ExportModelOperationsMixin("dokument"), ModelWithMetadata):
         null=True,
         related_name="dokument_historie",
     )
-    licence = models.TextField(blank=True, null=True)
+    licence = models.ForeignKey(
```

Comment:
Nevím, zda není opomenuto NOT NULL (logicky půjde nastavit až po tom, co se provede migrace samotná)